rspec-tools/rspec_template/multi_language/common/metadata.json

```
@@ -15,5 +15,13 @@
   "sqKey": "S${RSPEC_ID}",
   "scope": "All",
   "defaultQualityProfiles": ["Sonar way"],
-  "quickfix": "unknown"
+  "quickfix": "unknown",
+  "code": {
+    "impacts": {
+      "MAINTAINABILITY": "HIGH",
+      "RELIABILITY": "MEDIUM",
+      "SECURITY": "LOW"
+    },
+    "attribute": "FOCUSED"
```

```suggestion
    "attribute": "CONVENTIONAL"
```

`CONVENTIONAL` will be the default if nothing is specified. So I think it makes sense that we also use it here as the default. So if someone forgets to change it, the result is less odd.

---

Good point, thanks for mentioning that!